python/tskit/trees.py

```diff
@@ -2136,10 +2136,16 @@ def _minlex_postorder_traversal(self, u):
 
     def nodes(self, root=None, order="preorder"):
         """
-        Returns an iterator over the node IDs in this tree. If the root parameter
-        is provided, iterate over the node IDs in the subtree rooted at this
-        node. If this is None, iterate over all node IDs. If the order parameter
-        is provided, iterate over the nodes in required tree traversal order.
+        Returns an iterator over the node IDs reachable from the root(s) in this tree. If
+        the root parameter is provided, only the subtree rooted at this single root node
+        will be iterated over. If this parameter is None, iterate over the node IDs in
+        the subtrees rooted at each root node in turn. If the order parameter is
+        provided, iterate over the nodes in each subtree in the specified tree traversal
+        order.
+
+        .. note::
+            Not all nodes in the region spanned by this Tree may be
```

Comment:
Not sure this helps - people should be comfortable with the idea that most nodes are not "in" any particular tree. The confusing thing is that we can have *topology* that's not reachable from a root (if we have non-sample leaves that don't connect up with a rooted subtree). This is pretty obscure though.

Reply:
I've removed the note